chore: discourage AdministratorAccess policy (#12196)
aws/aws-cdk: packages/@aws-cdk/pipelines/README.md

> account only to bootstrap it and provision the initial pipeline. Otherwise,
> access to administrative credentials should be dropped as soon as possible.

> **On the use of AdministratorAccess**: The use of the `AdministratorAccess` policy
> by CDK Pipelines ensures that your pipeline can deploy every type of AWS resource

> > to your account, and can potentially make changes that affect your security posture.
> > Its use requires complete trust in the code and dependencies that make up your CDK app.
> > Check with the appropriate department within your organization to decide on the
> > proper policy to use.

Reviewer: Trying to make it a little less threatening, take it leave it...

 ```suggestion
> to your account. Make sure you trust all the code and dependencies that make up your CDK app..
> We recommend to check with the appropriate department within your organization to decide on the
> proper policy to use.
```

Author: Fair
